ontology/yaml/resources/METERS/entity_types/WM.yaml

```diff
@@ -36,3 +36,12 @@ WM_WF:
   - supply_water_volume_accumulator
   - water_linearvelocity_sensor
   - water_volume_accumulator
+
+WM_WVI:
```

Comment:
Please remove this type. The `fiveminutefixed_total_water_volume_sensor` is redundant because we could simply calculate that value from the `water_volume_accumulator` field. In general, fields that can be inferred or calculated from others are unnecessary.